maintainers/maintainer-list.nix

```diff
@@ -12390,4 +12390,11 @@
     github = "rski";
     githubId = 2960312;
   };
+  mbprtpmnr = {
```

Comment:
This file should be in a separate commit titled `maintainers: add mbprtpmnr`

Reply:
Solved!